File: backend/Race.Model/Models/Team.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Race.Model.Models
{
    public class Team
    {
        [Key]
        [Required]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime DateOfFoundation { get; set; }
        public string OwnerName { get; set; }
        public int ChampionShipPoints { get; set; }
        public virtual List<Pilot> Pilots { get; set; }
    }
}

File: frontend/src/app/teams/team.ts
import { IPilotViweModel } from '../pilots/IPilotViewModel';

export class ITeamViewModel {
    Id: number;
    Name: string;
    DateOfFoundation: string;
    OwnerName: string;
    ChampionShipPoints: number;
    Pilots: IPilotViweModel[];
}